app/schemas.py

```diff
@@ -0,0 +1,13 @@
+from marshmallow import Schema, fields
+
+
+class PostSchema(Schema):
+    body = fields.Str()
+    user_id = fields.Integer()
+    timestamp = fields.Date()
+
+
+class UserSchema(Schema):
+    username = fields.Str()
+    email = fields.Email()
+    posts = fields.Str
```

Comment:
`posts` is a nested field (an object inside another object). Looke here: https://marshmallow.readthedocs.io/en/stable/nesting.html